bower.json

```diff
@@ -1,4 +0,0 @@
-{
-  "name": "ember-page-title",
```

Comment:
shouldn't this get added to the package.json if it's removed from here?

Reply:
@MelSumner thanks for reviewing! I am not sure I understand. As in, should this go into the package.json's name field? I think it is already there: https://github.com/adopted-ember-addons/ember-page-title/blob/latest/package.json#L2
Or would this go into another field in package.json?

Reply:
I completely misread this change. Sorry for the confusing comment, please ignore.